<launch>

    <!-- Arguments -->
    <arg name="on_robot" default="false"/>

    <!-- Launch Realsense camera node with IMU feedback -->
    <include file="$(find-pkg-share realsense2_camera)/launch/rs_launch.py">
        <arg name="enable_gyro" value="true" />
        <arg name="enable_accel" value="true" />
    </include>

    <!-- Launch control_servos node if on_robot==true -->
    <node pkg="car_control" exec="control_servos" if="$(eval '\'$(var on_robot)\' == \'true\'')">
    </node>

    <!-- Launch odometry node -->
    <node pkg="car_control" exec="odometry">
    </node>

    <!-- Launch steering_control node -->
    <node pkg="car_control" exec="steering_control">
    </node>

</launch>